CHANGELOG.md

```diff
@@ -1,44 +1,3 @@
-# Changelog
-
-All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
-
-## [Unreleased]
```

Comment:
Are these removals of the unreleased intentional? I remember seeing this file in the new format, but I think that's gone now after some new commits.
🤔

Reply:
It is intentional, yes. The changelog is generated automatically, so there's no need for an unreleased section. In the first commits we were using a different library (standard-version) that required this section. Release-it works great without it.

Reply:
wait, now I get it. I'll release a new version in a sec, this will go away hahaha

Reply:
They'll be added once we merge this PR 

![](https://media0.giphy.com/media/ibVgJEtugfvVjqE3r6/giphy.gif?cid=ecf05e4717z2smd6yl390z09l6pwkwlr8dopqojx1tbsrwrb&rid=giphy.gif&ct=g)